convene-web/app/models/workspace.rb

```diff
@@ -4,6 +4,9 @@ class Workspace < ApplicationRecord
   # Which client owns the workspace
   belongs_to :client
 
+  # The fully-qualified domain to enter the workspace
+  attribute :branded_domain, :string
```

Comment:
Do we need to add some unique constraint/ validation on `branded_domain`?

Reply:
Good point! I've added a uniquness validation.